test/model/enc_frame.py

```diff
@@ -164,21 +166,95 @@ def zigzag(block):
         r[zig_zag_order[i]] = block[i]
     return r
 
-#def rle_code(block):
-#    entropy = []
-#    count = 0
-#    for i in block:
-#        if()
-
-#    return entropy
-
 def rle_code(block):
     return list(chain(*[(symbol,len(list(g))) for symbol, g in groupby(block)]))
 
+def build_huffman_rom_tables(csvfile):

```

Comment:
This function is defined again here....
`def build_huffman_rom_tables(csvfile):`